doc/Command-Reference.md

```diff
@@ -4079,4 +4079,35 @@ This command displays the routing policy that takes precedence over the other ro
 		Exit routemap
   ```
 
+**Syslog config commands 
+
+This sub-section of commands is used to add or remove the configured syslog servers.
+
+**config syslog add 
+
+This command is used to add a SYSLOG server to the syslog server list.  Note that more that one syslog server can be added in the device.
+
+- Usage: config syslog add <ip-address>
+- Example: 
+  ```
+  admin@str-s6000-acs-11:~$ sudo config syslog add 1.1.1.1
+  Syslog server 1.1.1.1 added to configuration
+  Restarting rsyslog-config service...
+  admin@str-s6000-acs-11:~$
+  ```
+
+**config syslog delete
```

Comment:
Surround entire command with `**` to make it bold to match style of rest of document (`**config syslog delete**`).

Reply:
fixed